kaldb/src/main/proto/trace.proto

```diff
@@ -18,9 +19,10 @@ import "google/protobuf/empty.proto";
 enum ValueType {
   STRING  = 0;
   BOOL    = 1;
+  INT32   = 5;
   INT64   = 2;
   FLOAT64 = 3;
-  BINARY  = 4;
+  FLOAT32 = 6;
```

Comment:
Since Lucene doesn't support 32 bit or 64 bit values, why do we need 32 bit values in the value type here?

Reply:
I think it does? https://lucene.apache.org/core/9_5_0/core/org/apache/lucene/document/Field.html

Reply:
Can you please point to the specific field type where 32 bit ints or floats are specially handled?